cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -214,6 +214,38 @@ spec:
     k8s-app: kube-dns
 `
 
+	// KubeDNSClusterRole is the kube-dns ClusterRole manifest
+	KubeDNSClusterRole = `
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kube-dns
```

Comment:
typically, components that set up roles/rolebindings namespace to their component, e.g. `kubeadm:kube-dns` (see the rolebinding names kubeadm sets up like "kubeadm:kubelet-bootstrap")

Reply:
@liggitt thanks for the comments. coredns uses system:coredns (https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/addons/dns/manifests.go#L313). Should we keep consistent with coredns on the format?